resqpy/time_series/any_time_series_base_resqpy.py

```diff
@@ -0,0 +1,212 @@
+"""Any time series base resqpy"""
```

Comment:
I would rather the module name did not include the inherited class, so any_time_series.py

Even better, like Connor I prefer all the submodules to be flagged as private by giving them file names starting with _, so _any_time_series.py